### PR TITLE
User info form

### LINKED ISSUE #2: User Info Form
Form for the user to input their information.

### PR DESCRIPTION
**Pull request by Tapiiri**
Form for user to input their info.

### CODE REVIEW

File: src/actions/submitUserInfoAction.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const submitUserInfoAction = (infoObject = {}) => dispatch => {
+  dispatch({
+    type: "SUBMIT_USER_INFO_FORM_ACTION",
```
Comment (VonLatvala) [inline src/actions/submitUserInfoAction.js]: These types should be defined as constants in `src/constants/actions.js`

File: src/actions/updateUserInfoFormAction.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export const updateUserInfoFormAction = (infoObject = {}) => dispatch => {
+  dispatch({
+    type: "UPDATE_USER_INFO_FORM_ACTION",
```
Comment (VonLatvala) [inline src/actions/updateUserInfoFormAction.js]: Same as above

File: src/components/UserForm.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { connect } from "react-redux";
+import "../App.css";
+import { submitUserInfoAction } from '../actions/submitUserInfoAction'
+import { updateUserInfoFormAction } from '../actions/updateUserInfoFormAction'
+
```
Comment (VonLatvala) [inline src/components/UserForm.js:6]: we need propTypes

File: src/reducers/userReducer.js [hunk 4]
```diff
@@ -5,6 +5,22 @@ export default (state = {}, action) => {
...
       };
+    case "UPDATE_USER_INFO_FORM_ACTION":
+      return {
+        ...state,
+        form: {
+          ...state.form,
+          ...action.payload
+        }
+      }
+    case "SUBMIT_USER_INFO_FORM_ACTION":
```
Comment (VonLatvala) [inline src/reducers/userReducer.js]: These to constants as said above

**Review by VonLatvala — CHANGES_REQUESTED** [review]
Some code quality 'issues', otherwise looks fine. Will run this when the code quality issues are resolved ;)

File: src/components/UserForm.js (hunk 3 above)
Comment (Tapiiri) [reply src/components/UserForm.js:6]: Good point - I'll fix it for the component, but I also now posted an Issue about it - https://github.com/TiK15Ryyb/etasitsit-frontend/issues/10

File: src/actions/submitUserInfoAction.js (hunk 1 above)
Comment (Tapiiri) [reply src/actions/submitUserInfoAction.js]: Done

Comment (Tapiiri) [reply src/actions/submitUserInfoAction.js]: https://github.com/TiK15Ryyb/etasitsit-frontend/blob/2005f48ca6e5615d22a202fcf195cd6e2b30b6c8/src/components/UserForm.js#L18-L25

https://github.com/TiK15Ryyb/etasitsit-frontend/blob/2005f48ca6e5615d22a202fcf195cd6e2b30b6c8/src/components/UserForm.js#L52-L55

File: src/components/UserForm.js (hunk 3 above)
Comment (Tapiiri) [reply src/components/UserForm.js:6]: https://github.com/TiK15Ryyb/etasitsit-frontend/blob/2005f48ca6e5615d22a202fcf195cd6e2b30b6c8/src/components/UserForm.js#L18-L25

https://github.com/TiK15Ryyb/etasitsit-frontend/blob/2005f48ca6e5615d22a202fcf195cd6e2b30b6c8/src/components/UserForm.js#L52-L55

Also fixed in same way for Table and Seat.

File: src/actions/submitUserInfoAction.js (hunk 1 above)
Comment (Tapiiri) [reply src/actions/submitUserInfoAction.js]: https://github.com/TiK15Ryyb/etasitsit-frontend/blob/422af1aac61a235f67cebfdb4902afc6130929bd/src/actions/updateUserInfoFormAction.js#L1-L2

https://github.com/TiK15Ryyb/etasitsit-frontend/blob/422af1aac61a235f67cebfdb4902afc6130929bd/src/constants/actions.js#L1-L3

File: src/components/UserForm.js (hunk 3 above)
Comment (Tapiiri) [reply src/components/UserForm.js:6]: Ended up reverting this and instead using the once done in devops branch.

**Comment by Tapiiri** [comment]
GitHub says here that there are conflicts to be resolved, but there's no problem when locally merging - must have gotten stuck to the phase before the master was merged in.

**Review by VonLatvala — APPROVED** [review]
nii-san~

**Pull request merged**